js/helpers/replaceAll.js

```diff
@@ -0,0 +1,17 @@
+var escapeRegExp = require( "lodash/string/escapeRegExp" );
+
+/**
+ * Replace placeholders in the passed string.
+ * @param {string} string The string that needs placeholders to be replaced.
+ * @param {object} mapObject The replacement values to be used.
+ * @returns {string} The string with the replaced placeholders.
+ */
+var replaceAll = function( string, mapObject ) {
```

Comment:
Calling a variable an `object` is very tempting, but it doesn't add anything to the name. Everything is an object in JavaScript.

`replacements` is better here.